config/deploy/stage.rb

```diff
@@ -0,0 +1,6 @@
+ENV['APP_ENV']  ||= 'staging'
+require_relative 'server_settings'
+
+server 'sdr-services-test.stanford.edu', user: 'sdr2service', roles: %w{app}
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
Why do we need a dev environment that doesn't behave like production?  Wouldn't that just be our laptop?